scripts/docker-build.sh

```diff
@@ -80,27 +76,82 @@ mkdir -p ${BUILD_DIR}/.gradle
 ln -sfn ${BUILD_DIR}/.gradle /root/.gradle
 
 pushd ${BUILD_DIR}
-cp ${SOURCE_DIR}/scripts/ci/config.pri ${SOURCE_DIR}/config.pri
-${QT_ANDROID}/bin/qmake ${SOURCE_DIR}/QField.pro "APP_NAME=${APP_NAME}"
-make
-make install INSTALL_ROOT=${INSTALL_DIR}
+
+export ANDROIDNDK=/opt/android-ndk
+export ANDROIDAPI=21
+if [ "X${ARCH}" == "Xx86" ]; then
+    export ANDROID_ARCH=x86
+    export SHORTARCH=x86
+elif [ "X${ARCH}" == "Xarmv7" ]; then
+    export ANDROID_ARCH=armeabi-v7a
+    export SHORTARCH=arm
+elif [ "X${ARCH}" == "Xarm64_v8a" ]; then
+    export ANDROID_ARCH=arm64-v8a
+    export SHORTARCH=arm64
+elif [ "X${ARCH}" == "Xx86_64" ]; then
+    export ANDROID_ARCH=x86_64
+    export SHORTARCH=x86_64
```

Comment:
can we move this to scripts/version_number.sh which is already called here?

Reply:
It's not really a "version_number" thing, more an "arch" thing.
I don't like this block at all actually.
I would love to have something more stable (just don't know what. yet). I'd prefer to keep it here as a reminder that it needs to be improved.